packages/vite/src/node/ssr/ssrTransform.ts

```diff
@@ -97,8 +97,8 @@ async function ssrTransformScript(
   function defineImport(node: Node, source: string) {
```

Comment:
The `node` parameter can be removed as it's not used anymore

Reply:
Done.